src/test/java/guitests/PanelRenameTest.java

```diff
@@ -130,4 +139,32 @@ public void panelRenameTest() {
         push(KeyCode.ENTER);
         sleep(EVENT_DELAY);
     }
+
+    @Test
+    public void panelRenameSelectOnlyOnePanel() {
```

Comment:
Try to use the new naming convention for test methods. It's given in the java coding standard.